packages/npm-package-json-lint-config/test/index.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import isPlainObj from 'is-plain-obj';
```

Comment:
Minor: Not sure if introduction of dependency here is strictly necessary. What is it we're testing? 

Would [`expect.toMatchObject( {} )`](https://facebook.github.io/jest/docs/en/expect.html#tomatchobjectobject) have been adequate, particularly if we don't care about subset of properties?

Alternatively, Lodash includes an [`_.isPlainObject`](https://lodash.com/docs/4.17.10#isPlainObject) as well, if we want to lean on Lodash as a canonical source of utility functions.

Thinking in terms of being explicit with where dependencies add value to avoid excessive proliferation.